srcpkgs/grub/template

```diff
@@ -141,19 +148,31 @@ grub-utils_package() {
 		vmove usr/bin/grub-mkfont
 	}
 }
-grub-x86_64-efi_package() {
+
+supplementary_package() {
+	_TARGET=${1}
+	_GRUB_TARGET=${2}
+	_PLATFORM=${3}
 	noarch=yes
 	depends="grub>=$version dosfstools efibootmgr"
-	short_desc+=" - x86_64 EFI support"
+	short_desc+=" - ${_TARGET} ${_PLATFORM} support"
 	pkg_install() {
-		vmove usr/lib/grub/x86_64-efi
+		vmove usr/lib/grub/${_GRUB_TARGET}-${_PLATFORM}
 	}
 }
+
+grub-i386-pc_package() {
+	supplementary_package i386 i386 pc
```

Comment:
please just be explicit and repeat the definition on each package

stop trying to be smart

Reply:
You can also stop using such a condescending tone.
I will update the PR to fulfill your desire.